### ISSUE TITLE
Menu squashed when opened

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#326]
```
What steps will reproduce the problem?

Whenever I open the game it opens in windowed mode (which is fine) but this 
messes up the options menu

What is the expected output? What do you see instead?

When I go into the options menu it is clearly designed for a higher resolution 
as the text bxex are squashed and you cannot see what is in the "Net nick", 
"Password" and "Locale" my desktop is runnng at 1920x1080

What version of the product are you using? On what operating system?

I am running "Hegewars 0.9.17" 

Please provide any additional information below.

My PC runs with Windows 7 Home Premium 64-bit. I have an NVidia GTX460 (driver 
8.17.12.7533 and Directx11)and an Intel E6750 Core 2 Duo Porcessor

```

-----
Original issue reported on code.google.com by `hornbys...@gmail.com` on 7 Dec 2011 at 3:16
* Merged into: #359

Attachments:
* [Options Menu.jpg](https://storage.googleapis.com/google-code-attachments/hedgewars/issue-326/comment-0/Options Menu.jpg)

**Comment by GoogleCodeExporter** [comment]
```
There is an existing GCI task that was started about a week ago to break config 
up into two tabs, with more complex options on the Advanced tab.
This would free up space on this screen.

However, the frontend currently targets a minimum of 800x600 *fullscreen* so 
you may find problems w/ many other areas, such as the game rooms.


```

-----
Original comment by `kyberneticist@gmail.com` on 7 Dec 2011 at 3:35

**Comment by GoogleCodeExporter** [comment]
```
... the reason I bring this up is that in the screenshot you gave, while net 
nick and password were indeed rather squashed (locale luckily pops out so it 
isn't quite as bad from usability), the overall area, fullscreened, would be 
680px.
Sooo, I'll retest that area at 600px once the GCI task is done and see if we 
can tweak it (perhaps bottom nav area at lower resolutions?), but really, 
you're pushing our minimum height requirements and might find chat and play 
area to be rather constrained.
Really we are going more for "usable" at 600px high :)

```

-----
Original comment by `kyberneticist@gmail.com` on 8 Dec 2011 at 4:25

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `unC0Rr` on 4 Sep 2012 at 8:05
* Changed state: **Duplicate**
